apps/daimo-mobile/src/logic/note.ts

```diff
@@ -115,5 +42,6 @@ export function useEphemeralSignature(
   return ephemeralSignature;
 }
 
+// TODO: remove after we migrate to EIP-7212 contract
```

Comment:
This is independent of the P256 verifier change, this is a valid dummy signature for [ECDSA.sol](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/utils/cryptography/ECDSA.sol)